### ISSUE TITLE
Top-down Tanks Redux - Kenney

### ISSUE THREAD

**Issue by Kruger77** [4ian/GDevelop#6542]
## Description

Hello guys, I have just finished packaging this asset pack by Kenney I hope it can be useful for you

- License: Creative Commons CC0
- Author: Kenney
- Link to the original website: https://kenney.nl/assets/top-down-tanks-redux
- Zip file: [Top-down Tanks Redux - Kenney.zip](https://github.com/4ian/GDevelop/files/15123192/Top-down.Tanks.Redux.-.Kenney.zip)

**Comment by D8H** [comment]
Thank you for submitting this packaging.

Please also attach the project you used to generate the GDO file. It will ease review.

**Comment by Kruger77** [comment]
Of course, it's not an empty project, but I've renamed the scene to TanksAssetPack and there's only the folder with the assets inside. There's also the scene of that other asset pack that I sent after this
[assets.zip](https://github.com/4ian/GDevelop/files/15213112/assets.zip)

**Comment by D8H** [comment]
Thank you for submitting a pack.

The asset store doesn't handle custom collision masks nor origin. An automatic collision mask is evaluated on the 1st frame of the 1st animation.

It means that:
- Tank barrels and bullets collision masks will be off on other animations so they should be 1 Sprite object for each kind.
- Explosion animations won't be centered correctly so the frames need to be extended to 128x128 pixels.
  - You can do it with Gimp for instance
    https://docs.gimp.org/2.10/en/gimp-image-resize.html

Point of views are not states as users won't toggle between them. Barrels and creates should be independent sprites.
A color variation is usually not a state either. Fences and barricades should be independent sprites too.

Sprite names should almost never be plural.

**Comment by Kruger77** [comment]
Ok, thanks for the feedback, I'll work on these things and re send the pack

**Comment by Kruger77** [comment]
[Asset Store - Top-down.Tanks.Redux.-.Kenney.zip](https://github.com/4ian/GDevelop/files/15335635/Asset.Store.-.Top-down.Tanks.Redux.-.Kenney.zip)
now should be good

**Comment by D8H** [comment]
The project resources are broken.

Please can you attach an archive with:
- a project folder containing the 2 explosions (only this if possible)
- a folder with the other PNG (only the files that must be included as still sprites and excluding the version without underline as they won't be included in the pack)

**Comment by Kruger77** [comment]
> The project resources are broken.
> 
> Please can you attach an archive with:
> - a project folder containing the 2 explosions (only this if possible)
> - a folder with the other PNG (only the files that must be included as still sprites and excluding the version without underline as they won't be included in the pack)

Maybe because I changed the name of the file? I'll send you another zip file next week

**Comment by Kruger77** [comment]
I'm sorry I sent you this file so late
[Asset Store - Top-down.Tanks.Redux.-.Kenney.zip](https://github.com/user-attachments/files/15876934/Asset.Store.-.Top-down.Tanks.Redux.-.Kenney.zip)
